spec/requests/api/arbitration_profile_spec.rb

```diff
@@ -30,7 +31,6 @@
       expect do
         run_post(arbitration_profiles_url, request_body)
       end.to change(ArbitrationProfile, :count).by(1)
-      expect(response).to have_http_status(:ok)
```

Comment:
Was there a reason behind removing these expectations on the status code here and below?